prbt_hardware_support/test/unittests/unittest_disable_speed_monitoring.cpp

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2020 Pilz GmbH & Co. KG
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <string>
+
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+
+#include <ros/ros.h>
+
+#include <std_srvs/SetBool.h>
+
+#include <pilz_testutils/async_test.h>
+
+namespace disable_speed_monitoring_tests
+{
+
+static const std::string SPEED_MONITOR_SERVICE_NAME
+  {"/prbt/manipulator_joint_trajectory_controller/monitor_cartesian_speed"};
+
+static const std::string SPEED_MONITOR_CALLBACK_EVENT {"speed_monitor_callback"};
+
+class SpeedMonitorServiceMock
+{
+public:
+  MOCK_METHOD2(callback, bool(std_srvs::SetBool::Request&, std_srvs::SetBool::Response&));
+};
+
+class DisableSpeedMonitoringTest : public testing::Test, public testing::AsyncTest
+{
+};
+
+/**
+ * @tests{speed_monitoring_without_operation_mode_support,
+ *  Tests the deactivation of the speed monitoring.
```

Comment:
Please write a little bit more docu. It is extremely difficult to understand what "object/code" exactly this test tries to test. Write something like: "Tests the deactivation of the speed monitoring via service call from `fake_operation_mode_setup.launch` file."

Reply:
Done.